Lib/test/test_urlparse.py

```diff
@@ -982,6 +982,8 @@ def test_splithost(self):
                          ('www.example.org:80', ''))
         self.assertEqual(splithost('/foo/bar/baz.html'),
                          (None, '/foo/bar/baz.html'))
```

Comment:
You misunderstood the issue here. The issue is the precedence of fragments and basic authentication info. I don't think there is a concept of a comment in URL.